src/markdown-pages/build-apps/customize-open-source-visualization.mdx

```diff
@@ -0,0 +1,1141 @@
+        {'FROM Transaction SELECT percentage(count(*), WHERE duration < 0.1)'}
+      </code>
+    </CardBody>
+  </Card>
+);
+```
+
+This will revert the bar corners to the default 90-degrees instead of being rounded. While your local server is running, it automatically recognizes changes to _index.js_. So, view your visualization in your browser to see the update:
+
+![Progress bar with straight corners](../../images/build-an-app/progress-bar-chart-no-corner-radius.png)
+
+Perfect! You cloned and updated the open source **Circular progress bar** visualization from the New Relic One catalog. The only thing left to do is publish your version to the catalog so your accounts can subscribe to it.
+
+</Step>
+
+<Step>
+
+Now that you're ready to publish your Nerdpack, stop your local server with `CTRL+C`.
+
+</Step>
+
+</Steps>
+
+## Add a custom visualization to a dashboard
+
+Publish your version of the **Victory charts** Nerdpack to the catalog. Then subscribe to it and use your visualization in a dashboard.
+
+<Callout variant="tip">
+
+Because you used `nr1 clone` to clone the Nerdpack's repository, your local copy already has its own UUID. This is a prerequisite for publishing your version to the New Relic One catalog. If you used `git clone` to copy, you need to update the Nerdpack's UUID manually:
+
+```sh
+nr1 nerdpack:uuid -gf
+[output] The new generated id is {purple}ab123c45-678d-9012-efg3-45hi6jkl7890
+```
+
+</Callout>
+
+<Steps>
+
+<Step>
+
+From its root directory, [publish](/build-apps/publish-deploy/publish/) your Nerdpack:
+
+```sh
+nr1 nerdpack:publish
+```
+
+</Step>
+
+<Step>
+
+[Subscribe](/build-apps/publish-deploy/subscribe/) to your Nerdpack:
+
+```sh
+nr1 nerdpack:subscribe
+```
+
+Here, you subscribed to your Nerdpack with the CLI. This is effectively the same action you performed earlier in this guide within the web UI, but from your terminal.
+
+</Step>
+
+<Step>
+
+Go to the **Apps** view in New Relic:
+
+![Navigate to apps](../../images/build-an-app/nav-to-apps.png)
+
+</Step>
+
+<Step>
+
+From **Apps**, open **Custom visualizations**:
+
+![Navigate to Custom visualizations](../../images/build-an-app/nav-to-custom-viz.png)
+
+From here, click the **Circular progress bar** visualization. Update your visualization's configuration options like you did when you were serving your Nerdpack locally.
+
+</Step>
+
+<Step>
+
+Click **Add to dashboard**:
+
+![Add to dashboard](../../images/build-an-app/add-to-dashboard.png)
+
+</Step>
+
+<Step>
+
+Go to your dashboard and see your new circular progress bar:
+
+![Circular progress bar in dashboard](../../images/build-an-app/circular-progress-bar-dashboard.png)
+
+</Step>
+
+</Steps>
+
+</Tutorial>
+
+## Summary
+
+In this guide, you:
+
+- Subscribed to a Nerdpack from the New Relic One catalog
+- Cloned an open source Nerdpack
+- Edited an existing visualization to meet your needs
+- Published and subscribed to your own custom Nerdpack
+- Added a Nerdpack to a dashboard
```

Comment:
We'll want to say added a visualization to the dashboard rather than the nerdpack. Maybe something like this?

"Added a visualization from your custom Nerdpack to a dashboard"